services/web-app/data/libraries.js

```diff
@@ -409,6 +417,15 @@ const devLibraries = {
     sponsor: 'carbon',
     group: carbonComponentsGroup
   },
+  'carbon-components-svelte': {
+    host: 'github.com',
+    org: 'carbon-design-system',
+    repo: 'carbon',
+    path: '/',
+    sponsor: 'carbon',
+    group: carbonComponentsGroup,
+    ref: '516-index-carbon-svelte'
+  },
```

Comment:
```suggestion
  'carbon-components-svelte': {
    host: 'github.com',
    org: 'carbon-design-system',
    repo: 'carbon-components-svelte',
    path: '/',
    sponsor: 'carbon',
    group: carbonComponentsGroup,
    ref: '516-index-carbon-svelte'
  },
```
@mattrosno does carbon sponsor svelte? just wanting to confirm

Reply:
Community maintained.

Reply:
@andreancardona got to use your org for now to test until we merge since it's a fork , also removing sponsor